DMB: Clarify and extend applications

cpaelzer opened the pull request:
Based on the most common questions that we got from applicants we extended on the list of expectations from Robie. Framing it more in the way an applicant would read and consume it. Extending by examples and details on the context.

Furthermore this split allows to add the often requested list of knowledge skills and how it maps to different application levels.

cpaelzer commented:
Right now - This is just the raw content improvement, I've not yet had a chance to check if the linking works and if the style renders correctly, but I wanted to bring it up as fast as possible for any related follow up work.

cpaelzer commented:
I have discussed with applicants, I have more to clarify and somewhat soften to not drive them away. On it as soon as I find time.

s-makin reviewed (commented):
Thanks for the re-write. I think overall it's a lot "friendlier" to frame the requirements as "how to make a good application" rather than what makes a bad one, so big +1 from me on that.

I've added a bunch of suggestions for clarity, plus some nits. The page with the table isn't in the RTD preview, but it's hard to read the table in markdown format (so I'll have to make any wording/polishing suggestions there in the next round).

cpaelzer commented on docs/staging/dmb/dmb-aspects-of-a-good-application.md:
```diff
@@ -0,0 +1,94 @@
+(dmb-aspects-of-a-good-application)=
+# DMB Aspects of a good application
```
This was meant to keep a common style of DMB being the prefix.
WDYT Sally, what wins - common style or sentence structure?

s-makin replied:
Sentence structure imo - the page titles can be played with, on all the DMB pages. The important thing is that they all contain dmb in the filename so we can set up ACLs.

cpaelzer replied:
ok

cpaelzer commented on docs/staging/dmb/dmb-aspects-of-a-good-application.md:
```diff
@@ -0,0 +1,94 @@
+
+Example:
+
+* Alice is working diligently on Openstack packages, getting many sponsored uploads every cycle. After a second cycle, in which only minimal / minor changes needed to happen, Alice feels confident in applying for cloud packageset upload rights.
```
yes that is known further outside as far as I can tell

cpaelzer commented on docs/staging/dmb/dmb-aspects-of-a-good-application.md:
```diff
@@ -0,0 +1,94 @@
+Topics must include endorsements from current Ubuntu members, developers, and uploaders.
+When applying for a specific level, having endorsements from current members with the same rights is required.
+For Canonical employees, it is recommended to have endorsements of members outside of Canonical.
+Depending on the upload rights being pursued, and breadth of delegated power, an applicant should have endorsements from outside their current Canonical organization.
```
True, I'll add that to my changes to add

cpaelzer commented on docs/staging/dmb/dmb-knowledge-requirements.md:
```diff
@@ -0,0 +1,54 @@
+(dmb-application-knowledge-requirements)=
```
yes, will do

cpaelzer replied:
@s-makin can you look again, this entry is in the index page already
```
grep -Hrn dmb-application-knowledge-requirements
index-dmb.md:34:dmb-application-knowledge-requirements
dmb-knowledge-requirements.md:1:(dmb-application-knowledge-requirements)=
dmb-aspects-of-a-good-application.md:77:We expect demonstrated understanding and evidence of such. An extended list is outlined at {ref}`DMB Application Knowledge Requirements <dmb-application-knowledge-requirements>`
```

cpaelzer commented on docs/staging/dmb/dmb-knowledge-requirements.md:
```diff
@@ -0,0 +1,54 @@
+There are several {ref}`DMB aspects of a good application <dmb-aspects-of-a-good-application>`, but questions often center around what knowledge and evidence an applicant should provide.
+
+Below are example topics along related questions that have been posed previously, along with a list of topics and the requisite knowledge
+Specific information depending on application tier is displayed in the appropriate columns in the table
```
hmm, ok  I think this is implied

cpaelzer commented on docs/staging/dmb/dmb-knowledge-requirements.md:
```diff
@@ -0,0 +1,54 @@
+Therefore, while contributing to the Ubuntu project, the focus should be on cultivating understanding rather than merely being able to provide answers to a predetermined set of questions
+Consider the following not as "questions you must be able to answer," but rather as "topics to comprehend, accompanied by a few illustrative questions that have arisen in the past."
+
+| Topic | Description => Expected level\*¹ | |  |  |  |  |
```
yes I'll move it up

cpaelzer replied:
done

cpaelzer commented on docs/staging/dmb/dmb-aspects-of-a-good-application.md:
```diff
@@ -0,0 +1,94 @@
+## Endorsements and Application Materials
+
+Applicants must follow the details of the {ref}`DMB application process <dmb-application>` to collect their materials on a page.
+The application must contain evidence of appropriate work for the level of upload rights, if you can also provide evidence of the learning topic along the way even better.
```
The evidence that proves that the stuff mentioned in the table is done/known.

cpaelzer commented:
I have added way more, in my todo before calling it ready for review again are
- elaborating on the reasons and limits of team vs !team endorsements 
- elaborating on what an endorsement can and have to cover and how to partially endorse without dragging someone down

cpaelzer commented on docs/staging/dmb/index-dmb.md:
```diff
@@ -30,7 +30,8 @@ ubuntu-developers
 dmb-agenda
-dmb-expectations
+dmb-aspects-of-a-good-application
+dmb-application-knowledge-requirements
```
That is it, I've renamed since the initial split.
Fixed now - thanks!

cpaelzer commented:
Rebased to remove conflicts with the wordlist and added the further topics I wanted to add.
@s-makin - ready for re-review by you.

Once you are happy I'll then bring it to the team for discussing the content itself.

s-makin reviewed (commented):
Some further suggestions :)

s-makin commented on docs/staging/dmb/dmb-aspects-of-a-good-application.md:
```diff
@@ -0,0 +1,118 @@
+When applying for a specific level, having endorsements from current members with the same or wider rights is required.
+
+It is recommended to have endorsements of members from outside your team/Organization, they provide implicit evidence of you being well connected in the community vs working in silo.
+Depending on the upload rights being pursued, and breadth of delegated power, this is more strict and those really should have endorsements from outside their current team and if possible company.
```
```suggestion
Depending on the upload rights being pursued, and breadth of delegated power, this is more strict and those really should have endorsements from outside their current team and, if possible, their company.
```

s-makin commented on docs/staging/dmb/dmb-aspects-of-a-good-application.md:
```diff
@@ -0,0 +1,118 @@
+
+It is recommended to have endorsements of members from outside your team/Organization, they provide implicit evidence of you being well connected in the community vs working in silo.
+Depending on the upload rights being pursued, and breadth of delegated power, this is more strict and those really should have endorsements from outside their current team and if possible company.
+To allow that to accrue while working towards upload rights, please consider to sometimes ask non-team or non-company members to review and sponsor your work.
```
```suggestion
To allow that to accrue while working towards upload rights, consider sometimes asking non-team or non-company members to review and sponsor your work.
```

s-makin commented on docs/staging/dmb/dmb-aspects-of-a-good-application.md:
```diff
@@ -0,0 +1,118 @@
+
+* Günther is working in the project for quite some time. They have 2 contributions to 3 packages each - and the people sponsoring those have endorsed Günther. But those have been long ago and there are two other sponsors that have done the majority of the work of Günther in recent years. Without an explanation that will make people question if that recent work was not good quality.
+
+* John wants to apply for the rather new debcrafter package set, but it only has one member so far. This is not a dead-lock due to requiring endorsements from the same level - endorsements by people sponsoring for him into other similar package sets or core developers would also back his case
```
```suggestion
* John wants to apply for the rather new debcrafter package set, but it only has one member so far. The requirement to have endorsements from the same level is not a blocker here -- endorsements by people sponsoring for him in other/similar package sets, or core developers, could also back his case.
```

s-makin commented on docs/staging/dmb/dmb-aspects-of-a-good-application.md:
```diff
@@ -0,0 +1,118 @@
+### Partial endoresement
+
+In the same context it is important to understand that people can partially endorse you.
+To be clear, please do not provide us with a puzzle of 27 fragmented partial endoresements - but a few can be helpful to fill some gaps.
```
```suggestion
To be clear, please do not provide us with a puzzle of 27 fragmented partial endorsements -- but a few can be helpful to fill some gaps.
```

s-makin commented on docs/staging/dmb/dmb-aspects-of-a-good-application.md:
```diff
@@ -0,0 +1,118 @@
+
+Example:
+
+* Paulina applies, has two full endoresments but also has worked with many more project members. 3 of them provide partial endorsements, one speaking about the kind interaction on bugs and the great chat behavior - another about great things they have sponsored - and another about a very complex transition you helped them. All of them mention that they do not cover the other aspects of a full end ...
```
```suggestion
* Paulina applies, has two full endorsements but also has worked with many more project members. Three of them provide partial endorsements: one speaking about the kind interaction on bugs and the great chat behavior; another about great things they have sponsored; and another about a very complex transition they helped with. All of them mention that they do not cover the other aspects of a full endorsement because they just didn't have this kind of exposure to Paulina yet.
```

s-makin commented on docs/staging/dmb/dmb-aspects-of-a-good-application.md:
```diff
@@ -0,0 +1,118 @@
+
+* Paulina applies, has two full endoresments but also has worked with many more project members. 3 of them provide partial endorsements, one speaking about the kind interaction on bugs and the great chat behavior - another about great things they have sponsored - and another about a very complex transition you helped them. All of them mention that they do not cover the other aspects of a full end ...
+
+* Ewan applies, has no full endorsement at all but three partial ones. The partial ones mention a somewhat positive aspect in regard to their work with you, but leave an ominous void about all else and thereby too much room for (negative) speculation.
```
```suggestion
* Ewan applies, has no full endorsement at all but three partial ones. The partial ones mention a somewhat positive aspect in regard to their work with him, but leave an ominous void about all else -- and thereby too much room for (negative) speculation.
```

s-makin commented on docs/staging/dmb/dmb-aspects-of-a-good-application.md:
```diff
@@ -0,0 +1,118 @@
+
+There are many things to know about, from a thorough understanding of (and experience in) Debian packaging, to Ubuntu-specific aspects such as Ubuntu package merges, SRUs, the release cycle, milestones and exceptions, autopkgtest and proposed migration, handling transitions, the operation of the seeds and MIRs.
+
+We expect demonstrated understanding and evidence of such. An extended list is outlined at {ref}`DMB Application Knowledge Requirements <dmb-application-knowledge-requirements>`
```
```suggestion
We expect demonstrated understanding and evidence of such. An extended list is outlined at {ref}`DMB Application Knowledge Requirements <dmb-application-knowledge-requirements>`.
```

s-makin reviewed (commented):
Just spotted a couple more nits, sorry!

cpaelzer commented on docs/staging/dmb/dmb-aspects-of-a-good-application.md:
```diff
@@ -0,0 +1,94 @@
+Please consider applying in steps, start with something easier to achieve matching your experience and contributions
+Then use these permissions to further collaborate and extend your track record to apply for the next stages sometime later.
+
+# Canonical employees
```
OK, accepted then

cpaelzer commented on docs/staging/dmb/dmb-application-knowledge-requirements.md:
```diff
@@ -0,0 +1,54 @@
+| To fork or not to fork | When adding a delta is appropriate, the maintenance burden of doing so, and how we track and manage that burden | I | A | B | A | A |
+| The possibility tree of versions and modifications | Understanding various exceptions to the basic derivative model, such as Ubuntu packaging a new upstream, syncing from experimental, SRUs and security updates as branches of the derivative graph | I | A | B | A | A |
+| Ubuntu and Launchpad packaging data model | Understanding the data model of packages, particularly within Launchpad: source and binary package versions being distinct, burning versions, the various pockets and pocket copies, source uploads only, one set of successful builds against each source upload only | I | I | B | A | A |
+|  |  |  |  |  |  |  |
```
No, that was an artifact of the conversion - removed

cpaelzer commented:
Addressed all feedback, rebased -  one open discussion remaining for now

cpaelzer commented:
Polished the elements reported by the spellcheck, which further made me standardize the mentioning of some groups/teams for consistency (but only int he DMB related section for now).

Rebased and ready for DMB team reivew so we can discuss and soften/harden the phrases where we think they need to change.

basak commented on docs/staging/dmb/dmb-aspects-of-a-good-application.md:
```diff
@@ -0,0 +1,120 @@
+
+Example:
+
+* Alice is working diligently on Openstack packages, getting many sponsored uploads every cycle. After a second cycle, in which only minimal / minor changes needed to happen, Alice feels confident in applying for cloud packageset upload rights.
```
"in which only minimal / minor *review* changes needed to happen" is I think what you mean, as opposed to packaging changes in general?

cpaelzer replied:
yes, "review triggered" is even more clear.
Thanks - added

basak commented on docs/staging/dmb/dmb-aspects-of-a-good-application.md:
```diff
@@ -0,0 +1,120 @@
+
+Example:
+
+* Evelyn is a member of the Canonical Foundations Toolchain Squad. They have worked extensively on Zig, not just on the toolchain but on many leaf packages. Over time, Evelyn has built up a portfolio of helping across Ubuntu, not just with Zig but in other areas. First, they applied for a speciality packageset `zig-packages`, and got endorsements from current packageset uploaders and a Ubuntu cor ...
```
Grammar: "shows a work"

cpaelzer replied:
Thanks for raising this - this was also a bit unclear still.
Changed it to what it really is:
"This demonstrates work and interaction within, but also outside their immediate team"

basak reviewed (commented):
This isn't a full review.

I very much welcome this restructuring - thank you! I suppose I agree with most of the material because it is based on my previously documented opinions :)

One point of contention - there has been a longstanding difference of opinion on the DMB on this point. What's required of a MOTU vs. Core Dev? I lean towards expecting the same standards of quality from both, except that a MOTU needs to know less about seeds and not really the MIR process at all. So I would want to push many of the core dev expectations column in your table over to MOTU.

I know others on the DMB have had different opinions on this in the past. I accept there are other view on this point.

cpaelzer commented on docs/staging/dmb/dmb-application.md:
```diff
@@ -50,7 +50,7 @@ membership. Here are some guidelines:
 To become a {ref}`ubuntu-developers-contributing`,
```
thanks, accepted

s-makin commented on docs/staging/dmb/dmb-application-knowledge-requirements.md:
```diff
@@ -0,0 +1,55 @@
+| Launchpad's bug model and Ubuntu bugs | How bug tasks work, community expectations from the bug tracker, team ownership and triage of bugs | B | I | IE | AE | AE |
+| Development Milestones | Understand the development milestones during the development cycle and what their purpose is.  | B | B |  | A | A |
+| Stable Point releases | Active releases will have point releases, which implies freezes, coordinated testing and more -- know your role and interaction with that. | B | B | A | A | A |
+| Freeze Exceptions (and how to get one) | Understand when an exception to a development freeze is appropriate and how to request one -- [Freeze Exception Process](https://wiki.ubuntu.com/FreezeExceptionProcess) | B | I | B | A | A |
```
```suggestion
| Freeze Exceptions (and how to get one) | Understand when an exception to a development freeze is appropriate and how to request one -- {ref}`Freeze Exception Process <freeze-exceptions>`. | B | I | B | A | A |
```

s-makin commented on docs/staging/dmb/dmb-application-knowledge-requirements.md:
```diff
@@ -0,0 +1,55 @@
+| *Library Transitions* | Understand how library transitions work in Ubuntu and how they can be tracked. | B | I |  | AE | AE |
+| *Operation of Seeds* | Understand the purposes of the Ubuntu Seeds and how to make a change to one -- {ref}`seed-management` | B | B |  | I | AE |
+| **Quality assurance** |  |  |  |  |  |  |
+| **Ubuntu Proposed Migrations** |  Understand that uploading or sponsoring is not the moment when you are done, what is {ref}`proposed-migration`, what value does it provide to the project and how to deal with the most common cases and issues. | B | I | I | A | A |
```
```suggestion
| **Ubuntu Proposed Migrations** |  Understand that uploading or sponsoring is not the moment when you are done; what is {ref}`proposed-migration` -- what value does it provide to the project and how to deal with the most common cases and issues. | B | I | I | A | A |
```

s-makin commented on docs/staging/dmb/dmb-application-knowledge-requirements.md:
```diff
@@ -0,0 +1,55 @@
+| **Active Releases** |  |  |  |  |  |  |
+| **Ubuntu Stable Release Update process** | Broad understanding of all [SRU types](https://canonical-sru-docs.readthedocs-hosted.com/en/latest) and how to drive them. Detailed understanding of the specifics required for a standard SRU. Able to identify the next step needed for any given SRU. | B | I | A | I | A |
+| Understand the Stable Release Update process | Understand the purpose of the Stable Release Update process and how it operates across released versions of Ubuntu and the development release. | B | I | A | A | A |
+| Upload and testing of stable release updates (SRU) | Demonstrate experience of stable release updates to packages in Ubuntu. Multiple uploads to the same are good, but the wider the application should allow uploads the more demonstrating some width is required. | BE | BE | IE | IE | AE |
```
```suggestion
| Upload and testing of stable release updates (SRU) | Demonstrate experience of stable release updates to packages in Ubuntu. Multiple uploads to the same package are good, but the wider the application should allow uploads, the more demonstrating some width is required. | BE | BE | IE | IE | AE |
```

s-makin commented on docs/staging/dmb/dmb-application-knowledge-requirements.md:
```diff
@@ -0,0 +1,55 @@
+| Understand the Stable Release Update process | Understand the purpose of the Stable Release Update process and how it operates across released versions of Ubuntu and the development release. | B | I | A | A | A |
+| Upload and testing of stable release updates (SRU) | Demonstrate experience of stable release updates to packages in Ubuntu. Multiple uploads to the same are good, but the wider the application should allow uploads the more demonstrating some width is required. | BE | BE | IE | IE | AE |
+| **Universe vs Main** |  |  |  |  |  |  |
+| **Main Inclusion Review (MIR) process** | Understand the {ref}`MIR process <main-inclusion-review>`, its purpose. When invoking that process is needed and how to do so. | B | I |  | B | A |
```
```suggestion
| **Main Inclusion Review (MIR) process** | Understand the {ref}`MIR process <main-inclusion-review>` and its purpose; When invoking the MIR process is needed, and how to do so. | B | I |  | B | A |
```

s-makin commented on docs/staging/dmb/dmb-application-knowledge-requirements.md:
```diff
@@ -0,0 +1,55 @@
+| Upload and testing of stable release updates (SRU) | Demonstrate experience of stable release updates to packages in Ubuntu. Multiple uploads to the same are good, but the wider the application should allow uploads the more demonstrating some width is required. | BE | BE | IE | IE | AE |
+| **Universe vs Main** |  |  |  |  |  |  |
+| **Main Inclusion Review (MIR) process** | Understand the {ref}`MIR process <main-inclusion-review>`, its purpose. When invoking that process is needed and how to do so. | B | I |  | B | A |
+| Understand main vs universe | Why does the [split exist](https://documentation.ubuntu.com/server/tutorial/managing-software/#where-do-packages-come-from%20), what does it mean to the Developer and what to the user | B | I | I | A | A |
```
```suggestion
| Understand main vs universe | Why does the [split exist](https://documentation.ubuntu.com/server/tutorial/managing-software/#where-do-packages-come-from%20)? What does it mean to the Developer? What does it mean to the user? | B | I | I | A | A |
```

s-makin commented on docs/staging/dmb/dmb-application-knowledge-requirements.md:
```diff
@@ -0,0 +1,55 @@
+| **Universe vs Main** |  |  |  |  |  |  |
+| **Main Inclusion Review (MIR) process** | Understand the {ref}`MIR process <main-inclusion-review>`, its purpose. When invoking that process is needed and how to do so. | B | I |  | B | A |
+| Understand main vs universe | Why does the [split exist](https://documentation.ubuntu.com/server/tutorial/managing-software/#where-do-packages-come-from%20), what does it mean to the Developer and what to the user | B | I | I | A | A |
+| MIR submission | Having submitted, and finalized MIR cases |  | BE |  |  | IE |
```
```suggestion
| MIR submission | Having submitted, and finalized, MIR cases. |  | BE |  |  | IE |
```

s-makin commented on docs/staging/dmb/dmb-application.md:
```diff
@@ -47,10 +47,10 @@ membership. Here are some guidelines:
 {ref}`Per-Package Uploader <ubuntu-developers-per-package>`,
-{ref}`Core Developer (core-dev) <ubuntu-developers-core-dev>`,
+{ref}`Ubuntu Core Developer (core-dev) <ubuntu-developers-core-dev>`,
 {ref}`SRU developers <ubuntu-developers-sru>` or
```
```suggestion
{ref}`SRU developer <ubuntu-developers-sru>` or
```

s-makin commented on docs/staging/dmb/dmb-application.md:
```diff
@@ -89,9 +89,18 @@ as visible under `https://launchpad.net/~<LPUSERNAME>/+codesofconduct`.
-is reached (4 of 7 currently), will add the applicant to the team or ask them to
-re-apply in due time.
+the team.
+During the meeting the DMB members will cast their votes.
```
```suggestion
During the meeting the DMB members will cast their votes.

```

s-makin commented on docs/staging/dmb/dmb-application-knowledge-requirements.md:
```diff
@@ -0,0 +1,55 @@
+|  |  | PPU*² | pkg-set | SRU-dev | MOTU | core-dev |
+| **Upstream/Downstream** |  |  |  |  |  |  |
+| The Derived Distribution Model | Understanding the derivative model (upstream \-\> Debian \-\> Ubuntu, how deltas are added and managed, basic package versioning in regard to that context. Ubuntu as a derivative) | I | I | B | A | A |
+| To fork or not to fork | When adding a delta is appropriate, the maintenance burden of doing so, and how we track and manage that burden | I | A | B | A | A |
```
```suggestion
| To fork or not to fork | When adding a delta is appropriate, the maintenance burden of doing so, and how we track and manage that burden. | I | A | B | A | A |
```

s-makin commented on docs/staging/dmb/dmb-application-knowledge-requirements.md:
```diff
@@ -0,0 +1,55 @@
+| To fork or not to fork | When adding a delta is appropriate, the maintenance burden of doing so, and how we track and manage that burden | I | A | B | A | A |
+| The possibility tree of versions and modifications | Understanding various exceptions to the basic derivative model, such as Ubuntu packaging a new upstream, syncing from experimental, SRUs and security updates as branches of the derivative graph | I | A | B | A | A |
+| Ubuntu and Launchpad packaging data model | Understanding the data model of packages, particularly within Launchpad: source and binary package versions being distinct, burning versions, the various pockets and pocket copies, source uploads only, one set of successful builds against each source upload only | I | I | B | A | A |
+| Differences between Debian and Ubuntu | Deviations from Debian such as no binNMUs (or any binary uploads), no individual package "ownership", team uploads only, management of transitions | B | B | B | A | A |
```
```suggestion
| Differences between Debian and Ubuntu | Deviations from Debian such as no binNMUs (or any binary uploads), no individual package "ownership", team uploads only, management of transitions. | B | B | B | A | A |
```

s-makin commented on docs/staging/dmb/dmb-application-knowledge-requirements.md:
```diff
@@ -0,0 +1,55 @@
+| Ubuntu and Launchpad packaging data model | Understanding the data model of packages, particularly within Launchpad: source and binary package versions being distinct, burning versions, the various pockets and pocket copies, source uploads only, one set of successful builds against each source upload only | I | I | B | A | A |
+| Differences between Debian and Ubuntu | Deviations from Debian such as no binNMUs (or any binary uploads), no individual package "ownership", team uploads only, management of transitions | B | B | B | A | A |
+| Ubuntu Code of Conduct and decision making | How decisions are made in Ubuntu: CC, TB, Ubuntu Developers at large, and the key privileged teams (SRU, Release, AA). Operating by consensus and how to handle disagreements. | B | B | B | A | A |
+| Canonical's interest in Ubuntu for Canonical employees | Separation of Canonical and Ubuntu and how to deal with decision making when they are conflicted | B | B | A | A | A |
```
```suggestion
| Canonical's interest in Ubuntu for Canonical employees | Separation of Canonical and Ubuntu and how to deal with decision making when they are conflicted. | B | B | A | A | A |
```

s-makin commented on docs/staging/dmb/dmb-application-knowledge-requirements.md:
```diff
@@ -0,0 +1,55 @@
+| Differences between Debian and Ubuntu | Deviations from Debian such as no binNMUs (or any binary uploads), no individual package "ownership", team uploads only, management of transitions | B | B | B | A | A |
+| Ubuntu Code of Conduct and decision making | How decisions are made in Ubuntu: CC, TB, Ubuntu Developers at large, and the key privileged teams (SRU, Release, AA). Operating by consensus and how to handle disagreements. | B | B | B | A | A |
+| Canonical's interest in Ubuntu for Canonical employees | Separation of Canonical and Ubuntu and how to deal with decision making when they are conflicted | B | B | A | A | A |
+| Canonical's interest in Ubuntu for everyone else | Separation of Canonical and Ubuntu and how to deal with decision making when they are conflicted | B | B | B | B | B |
```
```suggestion
| Canonical's interest in Ubuntu for everyone else | Separation of Canonical and Ubuntu and how to deal with decision making when they are conflicted. | B | B | B | B | B |
```

utkarsh2102 reviewed (changes requested):
hi! sorry to be a PITA, is it possible to wrap up in about 80 characters? @s-makin has mostly been doing that for most of the docs and I've been trying to do that as much also. 😅
